src/app/Http/Controllers/API/PresentationController.php

```diff
@@ -38,7 +39,27 @@ public function index(User $user)
      */
     public function store(Request $request, User $user)
     {
-        //
+        // check that the data is valid
+        $request->validate([
+            'title' => 'required',
+            'date' => 'required|date',
+        ]);
+
+        // data is valid
+        // create the new presentation
+        $presentation = Presentation::create($request->only([
+            'title',
+            'date'
+        ]));
+
+        // link the new presentation w/ it's presenter
+        // i.e the current user
+        $presentation->users()->attach($user, [
+            'role' => 'presenter'
+        ]);
```

Comment:
Here you may do something like `$user->presentations()->create($request->..., ['role' => 'presenter'])` not shure about the syntax concerning the pivot param but it will be better than create a presentation without any defined presenter and then assign it

Reply:
`ba62ba3 `